node/node.go

```diff
@@ -672,6 +677,12 @@ func decodeStakeCall(getData []byte) int64 {
 	value.SetBytes(getData[4:]) //Escape the method call.
 	return value.Int64()
 }
+
+func decodeFuncSign(data []byte) string {
+	funcSign := hexutil.Encode(data[:4])
```

Comment:
can you please show me the reference of this logic?

Reply:
This is based on a smart contract call, first 4 bytes are function call and later bytes are the params. Please refer to how `data` is generated for smart contracts in ethereum.

Reply:
that's why i need you to give the reference? why 4? you should have put 4 as a constant at the top and give the reason/reference why 4?

Reply:
Done.